app/styles/_base.scss

```diff
@@ -0,0 +1,86 @@
+// Border box all the things by default
+*,
+*:before,
+*:after {
+  box-sizing: border-box;
+}
+
+html {
+  background-color: $html-background-color;
+  height: 100%;
+}
+
+body {
+  color: $text-color;
+  @include font();
+  font-size: $base-font;
+  padding-bottom: 20px;
+}
+
+noscript {
+  color: $error-text-color;
+  display: block;
+  margin: 160px 0 0 0;
+}
+
+a {
+  color: $link-color;
+  cursor: pointer; // Use the correct cursor for anchors without an href
+
+  &:active {
+    outline: none;
+  }
+
+  &:focus {
+    outline: 1px dotted $link-color;
+  }
+}
+
+header {
+  h1,
+  h2,
+  h3 {
+    @include headerFont();
```

Comment:
Yeah, actually the linting isn't that bad. Mostly "Avoid using id selectors.", which I feel like we should turn off since we're all grown-ups. Same with "Avoid qualifying class selectors with an element."

Most of the rest of the lint noise was from the app/styles/_media_queries.scss file which hasn't been touched.